source/documentation/managing_apps/quotas.md

```diff
@@ -16,6 +16,20 @@ Your quota sets the following:
 
 + Routes: The number of routes available to your applications (hostname and domain pairs where an application that exposes a listening port can be reached).
```

Comment:
We are technically supposed to avoid passive voice. Is there a way to make the sentence active? Who or what reaches the application?

Reply:
I agree, but this is unrelated to this PR

Reply:
Captured, I'll fix that later https://github.com/alphagov/paas-tech-docs/issues/404